### PR TITLE
Modify Logic for Below-Top-Nav Ads

### PR DESCRIPTION
**Pull request by kelvin-chappell**
@kenlim @Calanthe @austinh7

### CODE REVIEW

File: common/app/views/main.scala.html [hunk 1]
```diff
@@ -15,12 +15,14 @@
...
         case _ => { true }
     },
-    Edition(request),
-    Edition(request).id == "UK" && (
-        (AppleAdNetworkFrontSwitch.isSwitchedOn && metaData.id == "uk") ||
-        (AppleAdCultureFrontSwitch.isSwitchedOn && metaData.id == "uk/culture")
-    )
-)) { case (showAdverts, edition, adBelowNav) =>
+    Edition(request)
+)) { case (showAdverts, edition) =>
```
Comment (kenlim) [inline common/app/views/main.scala.html]: Can we get rid of this strange bracket?

**Comment by kenlim** [comment]
:+1

**Comment by kenlim** [comment]
:+1:

File: common/app/views/main.scala.html [hunk 2]
```diff
@@ -43,7 +45,7 @@
             @fragments.header(metaData)
         </div>
 
-        @if(showAdverts && adBelowNav) {
```
Comment (Calanthe) [inline common/app/views/main.scala.html]: why now showAdverts?

Comment (kelvin-chappell) [inline common/app/views/main.scala.html]: I changed the logic so that `adBelowNav` can only be true if `showAdverts` is true.

**Pull request merged**